jenkins/master.sls

```diff
@@ -0,0 +1,21 @@
+{% from "jenkins/map.jinja" import jenkins with context %}
+
+create_ssh_directory_on_master:
+  file.directory:
+    - name: {{ jenkins.home }}/.ssh
+    - user: {{ jenkins.user }}
+    - group: {{ jenkins.group }}
+    - mode: 700
+
+# This is just a static key, during production dynamic key generation is
+# recommended
```

Comment:
@nmadhok how do I do that actually? I have not looked into that yet. I try to do that as my next step. I just did the static insecure key setup so that it at least works. Will definitely change to dynamic, secure key generation.